src/signals/incident-management/containers/IncidentDetail/components/SplitNotificationBar/index.js

```diff
@@ -1,8 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
+import { Button, themeColor } from '@datapunt/asc-ui';
+import styled from '@datapunt/asc-core';
```

Comment:
We should use our own version and not the version that comes from another dependency.